libqtile/window.py

```diff
@@ -1316,3 +1316,12 @@ def cmd_set_position(self, dx, dy, curx, cury):
                 self.group.layout.focused = index2
                 self.group.layoutAll()
                 break
+
+    def __getstate__(self):
```

Comment:
Where are the `__getstate__` and `__setstate__` functions used? I don't see where they'd be called